packages/libs/client/src/utils/getPactErrorCode.ts

```diff
@@ -0,0 +1,39 @@
+export type PactErrorCode =
+  | 'RECORD_NOT_FOUND'
+  | 'DEFPACT_COMPLETED'
+  | 'CANNOT_RESOLVE_MODULE'
+  | 'EMPTY_CODE'
+  | 'ERROR';
+
+export function getPactErrorCode(
```

Comment:
We should add some docs above this one with a description that tells the user that it works for pact4 and pact5